frontend/doc-site/04__Analyze Public Data/4_1__Hosted Tutorials.mdx

```diff
@@ -68,6 +68,4 @@ Cellxgene allows you to find marker genes between selected cell populations.
 
 <EmbeddedGoogleSlides src="https://docs.google.com/presentation/d/e/2PACX-1vSodPEAz7gO7SQvWY5t9BRV1aNahWBFpG2USoKDxTmnHIzk9sN_j3ISkOL5vxvsdrFOiO0eXlCnd6ru/embed?start=false&loop=false&delayms=3000" />
 
-N.B: In the hosted explorer present on the cellxgene data portal, the differential expression feature has a limit of 50,000 cells (i.e. the sum of the number of cells in group 1 and group 2 cannot exceed 50,000). You can workaround this by only selecting subsamples of large clusters. We are currently working on ways to scale this calculation in the hosted setting.
-
-N.B.: You can find more information[ here](https://docs.cellxgene.cziscience.com/explore-data/algorithms#differential-expression) about how our differential expression is calculated. In brief, we use a Welch's t-test. While we are aware that single cell data does not always meet the assumptions imposed by this test, we utilize it because it performs well at identifying the _most._
+Note: You can find more information [here](https://docs.cellxgene.cziscience.com/explore-data/algorithms#differential-expression) about how our differential expression is calculated. In brief, we use a Welch's t-test. While we are aware that single cell data does not always meet the assumptions imposed by this test, we utilize it because it performs well at identifying the _most_ differentially expressed genes, and this is what our feature returns.
```

Comment:
```suggestion
Note: You can find more information <NextLink href="/docs/04__Analyze%20Public%20Data/4_3__Algorithms%20and%20Ontologies#differential-expression">here</NextLink> about how our differential expression is calculated. In brief, we use a Welch's t-test. While we are aware that single cell data does not always meet the assumptions imposed by this test, we utilize it because it performs well at identifying the _most_ differentially expressed genes, and this is what our feature returns.
```